packages/fxa-event-broker/lib/selfUpdatingService/index.ts

```diff
@@ -34,9 +41,12 @@ abstract class SelfUpdatingService<T> {
     const result = await this.updateFunction();
     if (isError(result)) {
       this.logger.error('clientCapabilityService', { err: result });
-      throw result;
+      if (this.failOnStart) {
```

Comment:
Should the error in the updateService interval propagate an error too?

Reply:
That only logs in any case, so logging that it still can't fetch seems fine.